src/lib/y2network/interface_type.rb

```diff
@@ -111,6 +111,8 @@ def method_missing(method_name, *arguments, &block)
     WIRELESS = new(N_("Wireless"), "wlan")
     # Infiniband card
     INFINIBAND = new(N_("Infiniband"), "ib")
+    # Infiniband card child device. Used in IPoIB (IP-over-InfiniBand)
+    INFINIBANDCHILD = new(N_("Infiniband Child"), "ibchild")
```

Comment:
why not INFINIBAND_CHILD?

Reply:
can be ... I was not aware of any policy in naming